apps/inbound-mail/src/server/inbound-mail.service.spec.ts

```diff
@@ -65,7 +66,11 @@ describe('Inbound Mail Service', () => {
         _organizationId,
         _userId,
       };
-      await inboundMailService.inboundParseQueueService.add(jobId, jobData, _organizationId);
+      await inboundMailService.inboundParseQueueService.add({
+        name: jobId,
+        data: jobData as unknown as IInboundParseDataDto,
```

Comment:
Make for the test, IInboundParseDataDto contained a lot of attributes that needed to be added. for now, added casting as those attributes were not tested.